tests/integration/components/paper-tabs-test.js

```diff
@@ -196,6 +196,7 @@ test('should support content inside inline label type', function(assert) {
   assert.equal(contentText, 'content that!');
 });
 
+/*
```

Comment:
@panthony What you can do for tests that need a wormhole, is to add `<div id="paper-wormhole"></div>` before the component in the hbs block.